content/physical_properties/index.rst

```diff
@@ -0,0 +1,24 @@
+.. _physical_properties_index:
+
+Physical Properties
+*******************
+
+
+This section covers some of the basic information regarding rocks physical properties. 
+
+
+
+Contents:
+
+.. toctree::
+   :maxdepth: 1
+
+   physical_properties_density
+   ../magnetics/magnetics_susceptibility
+   ../seismic/refraction/seismic_velocity
+   physical_properties_dielectric_permittivity
+   physical_properties_conductivity
+   ../induced_polarization/induced_polarization_physical_properties
```

Comment:
@rowanc1 : do you know if there is a better way to do this? b/c now, when you click on one of the pathed ones, it shows the true path on the navbar (ie. if you click on seismic velocity it puts you under seismic refraction instead of phys props, which could be confusing to students).